website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,315 @@
+| `consul.raft.apply`      | This counts the number of Raft transactions occurring over the interval. |
+| `consul.raft.commitTime` | This measures the time it takes to commit a new entry to the Raft log on the leader. |
+
+**Why they're important:** Taken together, these metrics indicate how long it takes to complete write operations
+in various parts of the Consul cluster. Generally these should all be fairly consistent and no more than a few
+milliseconds. Sudden changes in any of the timing values could be due to unexpected load on the Consul servers, or
+due to problems on the servers themselves.
+
+**What to look for:** Deviations (in any of these metrics) of more than 50% from baseline over the previous hour.
+
+### Leadership changes
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.raft.leader.lastContact` | Measures the time since the leader was last able to contact the follower nodes when checking its leader lease. |
+| `consul.raft.state.candidate` | This increments whenever a Consul server starts an election. |
+| `consul.raft.state.leader` | This increments whenever a Consul server becomes a leader. |
+
+**Why they're important:** Normally, your Consul cluster should have a stable leader. If there are frequent elections or leadership changes, it would likely indicate network issues between the Consul servers, or that the Consul servers themselves are unable to keep up with the load.
+
+**What to look for:** If `candidate` > 0, or `leader` > 0, or `lastContact` greater than 200ms.
+
+### Autopilot
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.autopilot.healthy` | This tracks the overall health of the local server cluster. If all servers are considered healthy by Autopilot, this will be set to 1. If any are unhealthy, this will be 0. |
+
+**Why it's important:** Obviously, you want your cluster to be healthy.
+
+**What to look for:** Alert if `healthy` is 0.
+
+### Memory usage
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.runtime.alloc_bytes` | This measures the number of bytes allocated by the Consul process. |
+| `consul.runtime.sys_bytes`   | This is the total number of bytes of memory obtained from the OS.  |
+| `mem.total`                  | Total amount of physical memory (RAM) available on the server.     |
+| `mem.used_percent`           | Percentage of physical memory in use. |
+| `swap.used_percent`          | Percentage of swap space in use. |
+
+**Why they're important:** Consul keeps all of its data in memory. If Consul consumes all available memory, it will crash. You should also monitor total available RAM to make sure some RAM is available for other processes, and swap usage should remain at 0% for best performance.
+
+**What to look for:** If `sys_bytes` exceeds 90% of `total_bytes`, if `mem.used_percent` is over 90%, or if
+`swap.used_percent` is greater than 0.
```

Comment:
Swap advice is potentially misleading - In general it's arguably better to disable swap entirely, but assuming they didn't the default `vm.swapiness` value in the kernel (at least on Ubuntu) is 60 which means some level of swapping will occur significantly before all available memory is exhausted.

Very small amounts of swap are expected and normal on any linux system with default swapinnes and more than a trivial amount of ram used. This is all subtle stuff to go into but it might be better to avoid mentioning it or onnly mention it being high not > 0